patches/l10n/adding-java.lsp.server-ja-and-zh_CN.diff

```diff
@@ -1,9 +1,158 @@
+diff --git a/netbeans-l10n-zip/src/ja/java/java-lsp-server/java-lsp-server/org/netbeans/modules/java/lsp/server/Bundle_ja.properties b/netbeans-l10n-zip/src/ja/java/java-lsp-server/java-lsp-server/org/netbeans/modules/java/lsp/server/Bundle_ja.properties
+new file mode 100755
+index 000000000..ab2edb02e
+--- /dev/null
++++ b/netbeans-l10n-zip/src/ja/java/java-lsp-server/java-lsp-server/org/netbeans/modules/java/lsp/server/Bundle_ja.properties
+@@ -0,0 +1,20 @@
++# Licensed to the Apache Software Foundation (ASF) under one
++# or more contributor license agreements.  See the NOTICE file
++# distributed with this work for additional information
++# regarding copyright ownership.  The ASF licenses this file
++# to you under the Apache License, Version 2.0 (the
++# "License"); you may not use this file except in compliance
++# with the License.  You may obtain a copy of the License at
++#
++#   http://www.apache.org/licenses/LICENSE-2.0
++#
++# Unless required by applicable law or agreed to in writing,
++# software distributed under the License is distributed on an
++# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
++# KIND, either express or implied.  See the License for the
++# specific language governing permissions and limitations
++# under the License.
++DESC_StartJavaDebugAdapterServer=Java\u30C7\u30D0\u30C3\u30B0\u30FB\u30A2\u30C0\u30D7\u30BF\u30FB\u30B5\u30FC\u30D0\u30FC\u3092\u958B\u59CB\u3057\u307E\u3059
++DESC_StartJavaLanguageServer=Java\u8A00\u8A9E\u30B5\u30FC\u30D0\u30FC\u3092\u958B\u59CB\u3057\u307E\u3059
++MSG_ConnectionSpecError=''{0}''\u3092\u63A5\u7D9A\u3068\u3057\u3066\u89E3\u6790\u3067\u304D\u307E\u305B\u3093\u3002''stdio''\u3001''listen:<port>''\u307E\u305F\u306F''connect:<port>''\u3092\u4F7F\u7528\u3057\u307E\u3059
```

Comment:
It seems odd to have only `'studio'` as an English word here. I think we should maintain *Visual Studio Code* or *VS Code* or any shorter variant as English text since it's a proper noun. 

On the other hand, even though *listen* and *connect* are technical terms, they may have technical translations which should be used.

Reply:
this is standard output (stdio) @sid-srini  , we can ignore this bit 
but there are extra quotes` MSG_ConnectionSpecError=Cannot parse '{0}' as connection. Use 'stdio', 'listen:<port>' or 'connect:<port>'`

i will manually correct them and raise issue with l10n team

Reply:
> this is standard output (stdio) @sid-srini , we can ignore this bit
> 

oops... thank you.